### PR TITLE
Request styles + footer

### PR DESCRIPTION
**Pull request by mschmele**
If you're okay with this merge it after the other one that I have open. It's mostly just styling the request form and adding a footer

### CODE REVIEW

File: app/assets/stylesheets/welcome.scss [hunk 1]
```diff
@@ -1,50 +1,90 @@
...
-	width: 500px;
-	height: 150px;
-	text-align: left;
-	margin-top: 10px;
-	margin-left: 168px;
+@import "common";
+
+.hero {
+	margin-top: -16px;
+	background: url("http://mumm.millersville.edu/files/4513/9794/7403/Screen_Shot_2014-04-19_at_6.43.22_PM.png");
```
Comment (mschmele) [inline app/assets/stylesheets/welcome.scss]: I wasn't sure what the path to the image was here so I just didn't bother to do that but I'll fix this eventually

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -1,18 +1,11 @@
...
-//
-// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
-// about supported directives.
-//
-//= require jquery
-//= require jquery_ujs
-//= require turbolinks
-//= require_tree .
-//= require jquery.validate
-//= require jquery.validate.additional-methods
```
Comment (kpollich) [inline app/assets/javascripts/application.js]: These JS requires need to exist or the site will break.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
   <title>WIXQ 91.7 :: Millersville University Radio</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```
Comment (kpollich) [inline app/views/layouts/application.html.erb]: Jquery is already included in `application.js` but you've removed that and replaced it with a CDN call we don't need.

**Comment by kpollich** [comment]
GitHub is saying you have conflicts to resolve. I've pushed to master a few time since this was filed. Could you pull and resolve anything that's conflicting?

**Comment by mschmele** [comment]
Yeah I'm ON IT

**Comment by mschmele** [comment]
:100: :100: :8ball: : :gun: :100: :100:

**Pull request closed without merging**